man/vi.1

```diff
@@ -1809,7 +1809,8 @@ Display buffers, Cscope connections, screens or tags.
 .Op Ar +cmd
 .Op Ar file
 .Xc
-Edit a different file.
+Edit a different file. If an initial capital letter is given the current
+screen is split horizontally.
```

Comment:
Suggest to phrase it as "The capitalized variant of the command splits the current screen to fit the new one." -- I think that's the intention. Once people tried, they will know the result.

Reply:
Thanks, I iterated the wording a bit.